doc/user/content/sql/create-sink/kafka.md

```diff
@@ -62,6 +62,7 @@ Field                | Value  | Description
 ---------------------|--------|------------
 `AVRO KEY FULLNAME`  | `text` | Default: `row`. Sets the Avro fullname on the generated key schema, if a `KEY` is specified. When used, a value must be specified for `AVRO VALUE FULLNAME`.
 `AVRO VALUE FULLNAME`| `text` | Default: `envelope`. Sets the Avro fullname on the generated value schema. When `KEY` is specified, `AVRO KEY FULLNAME` must additionally be specified.
+`NULL DEFAULTS`      | `bool` | Default: `false`. Whether to automatically have null defaults for nullable fields in the generated schemas.
```

Comment:
```suggestion
`NULL DEFAULTS`      | `bool` | Default: `false`. Whether to automatically default nullable fields to `NULL` in the generated schemas.
```

Reply:
Nit, `null` should be lowercase in the prose description ("Whether to automatically default nullable fields to `null` in the generated schemas.") because we are talking about json nulls, not SQL nulls.

Reply:
Updated to lowercase `null`